### ISSUE TITLE
Clicking on an item in the tree will cause the UI to scroll to a previous section

### ISSUE THREAD

**Issue by sbanni** [KirillOsenkov/MSBuildStructuredLog#290]
I have not been able to figure out exactly what the repro steps are for this but sometimes if I have a large list of items and then I scroll around the tree some and then try to click on something else the UI will scroll back to the large list instead of retaining focus where I was.

**Comment by KirillOsenkov** [comment]
Yes, I've seen this sometimes and it's annoying and would be nice to fix at some point.
